RELEASING.md

```diff
@@ -14,5 +14,22 @@ kpt-functions-catalog repo.
 1. Click `Publish release` button.
 1. Send an announcement email in the [kpt users google group].
 
+## Updating function docs
```

Comment:
To follow our own codelab convention, can you use an example to guide through the steps?
e.g.

The RELEASE_BRANCH branch should already exist in repo https://github.com/GoogleContainerTools/kpt-functions-catalog. It is in the form of ${FUNCTION_NAME}/v${MAJOR_VERSION}.${MINOR_VERSION}. For example, `set-namespace/v0.3` `kubeval/v0.1`

Reply:
Done